helsingborg-stad/app-mitt-helsingborg
# Feat/update react native to version 0.68.2

## Explain the changes you’ve made
Updated react-native to version 0.68.2

## Explain why these changes are made
The application falls behind on react and react-native version hence updating them.

## Explain your solution
Updated the react-native dependency and added/removed files that were not needed anymore. Updated dependencies that weren't compatible with the newer version of react-native.

## How to test

1. Checkout this branch
2. Make sure you clean the application first by running `yarn clean`
3.  Fire upp the simulator by running the command `yarn ios` or `yarn android`
4. The application should run as before

## Tested environments

- [] Storybook on a iOS device/simulator.
